lang/python/wiredtiger.i

```diff
@@ -652,6 +653,35 @@ OVERRIDE_METHOD(__wt_cursor, WT_CURSOR, search_near, (self))
 		$result = SWIG_FromCharPtr($1);
 }
 
+/* Handle returned wt_timestamp_t timestamps. */
+%typemap(in,numinputs=0) (wt_timestamp_t *timestamp) (uint64_t ts) { $1 = &ts; }
+%typemap(frearg) (wt_timestamp_t *timestamp) "";
+%typemap(argout) (wt_timestamp_t *timestamp) { $result = PyLong_FromUnsignedLongLong(*$1); }
+
+/* 
```

Comment:
In this comment, I would say that any numeric type (int or long in Py2, or int in Py3) is allowed as input for a timestamp.

Reply:
done